### PR TITLE
bubblewrap: new package

### PR DESCRIPTION
**Pull request by jdolitsky**
Depends on #33 being released (libcap)

### CODE REVIEW

File: bubblewrap.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - runs: |
+      meson \
+        --prefix=/usr \
+        --sysconfdir=/etc \
+        --mandir=/usr/share/man \
+        --localstatedir=/var \
+        --buildtype=plain \
+        -Drequire_userns=true . output
+      meson compile -C output
+      DESTDIR="${{targets.destdir}}" meson install --no-rebuild -C output
```
Comment (kaniini) [inline bubblewrap.yaml:44]: Should have docs subpackage, right?  It generates a manpage I think.

Comment (jdolitsky) [reply bubblewrap.yaml:44]: no, no manpages:
```
$ tar -xvf packages/x86_64/bubblewrap-0.6.2-r0.apk
x .PKGINFO
x usr
x usr/bin
x usr/bin/bwrap
x usr/share
x usr/share/bash-completion
x usr/share/bash-completion/completions
x usr/share/bash-completion/completions/bwrap
x usr/share/zsh
x usr/share/zsh/site-functions
x usr/share/zsh/site-functions/_bwrap
```

Comment (jdolitsky) [reply bubblewrap.yaml:44]: the meson command here is gleaned from the alpine build which uses `abuild-meson`, which appears to do just the following:
https://gitlab.alpinelinux.org/alpine/abuild/-/merge_requests/60/diffs

Comment (jdolitsky) [reply bubblewrap.yaml:44]: maybe since we dont have the `docbook-xsl` dependency?

Comment (jdolitsky) [reply bubblewrap.yaml:44]: https://git.alpinelinux.org/aports/tree/main/bubblewrap/APKBUILD#n11

Comment (kaniini) [reply bubblewrap.yaml:38]: oh, right, i didn't package any of that stuff because it was annoying.

**Pull request merged**